astro/exec2/process.go

```diff
@@ -22,9 +22,11 @@ import (
 	"io"
 	"os"
 	"os/exec"
+	"os/signal"
 	"syscall"
 	"time"
 
+	"github.com/hashicorp/go-multierror"
```

Comment:
fwiw, our https://godoc.org/go.uber.org/multierr would be a bit simpler to use than hashicorp's for the use cases below

Reply:
Hashicorp 'multierror' library is already used in this project (in other files), so I'm not sure if it's ok to introduce the library you mentioned. However, I've created a small commit [here](https://github.com/mdsjip/astro/commit/851eb8c001ce554b899f19fbebc89b7e525679d3) with it so I'd like to hear more opinions if we should use it.

IMO this can be done in a dedicated refactoring PR.

cc @dansimau @btromanova